app/controllers/my_facilities/ranked_facilities_controller.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+class MyFacilities::RankedFacilitiesController < AdminController
```

Comment:
What prompted a new controller for this? There's only one action and it seems like it could fit well within `MyFacilitiesController` no? It's duplicating a bunch of setup code, which would mean changing it in two places whenever we do so.

Reply:
It's primarily because I'm using `Reports::RegionService` which requires different setup than the My Facilities queries. I wanted to use exactly the same data that we're showing in reports.